cmd/local-admin-client.go

```diff
@@ -74,6 +74,14 @@ func (lc localAdminClient) ServerInfo() (sid ServerInfoData, e error) {
 	}, nil
 }
 
+// TopLocks - Returns the locks open on this server.
+func (lc localAdminClient) TopLocks() (LocksMap, error) {
+	if globalBootTime.IsZero() || globalLockServer == nil {
+		return LocksMap{}, errServerNotInitialized
+	}
+	return LocksMap{LockMap: globalLockServer.ll.lockMap}, nil
```

Comment:
note that caller is going to access globalLockServer.ll.lockMap without holding locks
maybe you can do:
`return globalLockServer.ll.lockMap.Dup(), nil`

Reply:
done